draft-bonaventure-mptcp-converters.mkd

```diff
@@ -431,13 +431,24 @@ Client                   Converter                       Server
 
 ## Sample Example of Incoming Converter-Assisted Multipath TCP Connection
 
-An example of an incoming converter-assisted Multipath TCP connection is depicted in {{fig-inestab}}. In order to support incoming connections from remote hosts, the client may use PCP {{RFC6887}} to instruct the converter to create dynamic mappings. Those mappings will be used by the converter to intercept an incoming TCP connection destined to the client and convert it into a Multipath TCP connection. 
+Remote Hosts can use the Transport Converter to connect to another Host.  We
+come across this situation typically when the Host is present in a private
+network and is not  directly accessible from outside. The connection
+establishment follows the steps as depicted in {{fig-inestab}}. Similar to
+{{sec-oconn}}, this document assumes that the Remote Host is configured with the
+set of Transport Converters and the information about the end Host to which the
+connection is initiated. This document also assumes that the access controls to
```

Comment:
Disagree. It is up to the internal host to instruct the converter whether it can accept incoming connections or not.